gtk4/src/lib.rs

```diff
@@ -17,6 +17,10 @@ extern crate libc;
 #[macro_use]
 extern crate bitflags;
 extern crate once_cell;
+#[doc(hidden)]
+#[allow(unused_imports)]
+#[macro_use]
+pub extern crate field_offset;
 
```

Comment:
You also want to re-export the `gtk4-macros` macros from here, just like it's done in `glib` with `glib-macros`